tests/Behat/RdfEntityContext.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content\Behat;
+
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Defines step definitions specifically for testing the RDF entities.
+ *
+ * This is needed due to issues with the Kernel testing of the RDF entity module
```

Comment:
Move to the test itself. The class is fine, it's just the test that is created to avoid the 8.6 issue.